File: app/src/main/java/can/siempredelao/f1kotlin/backend/MRDataDrivers.java
package can.siempredelao.f1kotlin.backend;

import com.google.gson.annotations.SerializedName;

public class MRDataDrivers {

    public String      xmlns;
    public String      series;
    public String      url;
    public String      limit;
    public String      offset;
    public String      total;
    @SerializedName("DriverTable")
    public DriverTable driverTable;
}

File: app/src/main/kotlin/can/siempredelao/f1kotlin/backend/model/MRDataDrivers.kt
package can.siempredelao.f1kotlin.backend.model

import com.google.gson.annotations.SerializedName

/** Collection of drivers */
class MRDataDrivers(val xmlns: String,
                    val series: String,
                    val url: String,
                    val limit: String,
                    val offset: String,
                    val total: String,
                    @SerializedName("DriverTable") val driverTable: DriverTable)
